src/main/webapp/pipe.js

```diff
@@ -308,11 +308,12 @@ function checkDataForError(data, errrorDivId) {
     }
 }
 
-function isTaskLoggedToConsole(task) {
-    return task.status.success
+function isTaskLoggedToConsole(data, task) {
```

Comment:
How about renaming this function to something like "isTaskLinkedToConsoleLog"? I think the proposed name is a bit misleading.